xbmc/utils/DiscsUtils.cpp

```diff
@@ -66,3 +75,25 @@ UTILS::DISCS::DiscInfo UTILS::DISCS::ProbeBlurayDiscInfo(const std::string& medi
 #endif
   return info;
 }
+
+bool UTILS::DISCS::IsBlurayDiscImage(const CFileItem& item)
+{
+  if (!item.IsDiscImage())
+    return false;
+
+  static constexpr std::array<const char*, 4> blurayFiles = {
```

Comment:
`std::array<std::string_view, 4>` makes creation of the `std::string` below a little more efficient.